tsconfig.json

```diff
@@ -21,6 +21,7 @@
     "experimentalDecorators": true,
     "jsx": "react",
     "baseUrl": ".",
+    "types": ["cypress"],
```

Comment:
can you please revert this change. This is not in scope of this pr also it can other side effects. https://www.typescriptlang.org/tsconfig#types
PS `e2e` have tsconfig too

Reply:
Done 💯